### ISSUE TITLE
Change git diffstat to (added, deleted) from (added, modified, deleted)

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#39390]
The Sourcegraph diffstat has historically been a little different than most code hosts render them: We have (added, modified, deleted), while most code hosts I know just have (added, deleted) and so does git, too. Our detection for “changed” is also pretty basic, it literally just counts - followed by +, so a - - + + diff replacing two lines would still be (2 added 0 modified 2 deleted). I think it would make things simpler and easier to understand if we dropped this concept of “changed” lines, which is underdefined anyways. With that we don’t have to rely on our very specific home-made library to parse (not that I want to switch but also we couldn’t) and we can likely take some short paths that save us a parse patch cycle, but mostly it just removes this confusing state. Git to me is red and green lines and that’s it.

**Comment by eseliger** [comment]
We will work on this _now_, but only drop the field from the API for 4.0, so after Aug 17.

**Comment by courier-new** [comment]
Do we have a follow-up issue filed to swap out our home-made library, now that that will be unblocked?
